plugins/amazonq/chat/jetbrains-community/resources/META-INF/plugin-chat.xml

```diff
@@ -34,7 +34,9 @@
             <reference id="aws.toolkit.open.q.window"/>
         </group>
         <!-- TODO: q.openchat will eventually be in amazonq, aws.toolkit.q.sign.in will eventually be in core. -->
-        <action id="q.openchat" class="software.aws.toolkits.jetbrains.services.amazonq.QOpenPanelAction"/>
+        <action id="q.openchat" class="software.aws.toolkits.jetbrains.services.amazonq.QOpenPanelAction">
+            <keyboard-shortcut keymap="$default" first-keystroke="meta alt Q" />
```

Comment:
This keystroke closes the Intellij IDE on Mac. I tried locally setting my open Q-chat keybind to this, but the IDE was still closing instead of opening chat.